cli/cmd/inject_test.go

```diff
@@ -45,4 +45,24 @@ func TestInjectYAML(t *testing.T) {
 			diffCompare(t, actualOutput, expectedOutput)
 		})
 	}
+
```

Comment:
Another nit: There's no need for this test, the way it is structured, to be in `TestInjectYAML`. It should be in a function `TestRunInjectCmd`.